spec/features/restaurant_spec.rb

```diff
@@ -115,7 +115,7 @@
     sign_in restaurant.user
     visit restaurant_path(restaurant)
 
-    expect(page).to have_content("Edit restaurant")
+    page.should have_selector(:link_or_button, 'Edit restaurant')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.